pkg/gatewayserver/io/io.go

```diff
@@ -159,6 +159,11 @@ func (c *Connection) HandleUp(up *ttnpb.UplinkMessage) error {
 		}
 		md.UplinkToken = buf
 		md.DownlinkPathConstraint = c.gateway.DownlinkPathConstraint
+
+		if len(c.gateway.Antennas) > int(md.AntennaIndex) {
+			location := c.gateway.Antennas[md.AntennaIndex].Location
+			md.Location = &location
```

Comment:
Ineffective variable

Reply:
Purpose is to pass a copy of the antenna location value. Is it safe to just pass a reference?